packages/pluggableWidgets/notifications-native/src/Notifications.tsx

```diff
@@ -109,7 +128,16 @@ export function Notifications(props: NotificationsProps<undefined>): null {
                 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                 // @ts-expect-error - see comment top top of file
                 onNotification(notification: IPushNotification) {
+                    let messageId = "";
+                    if ("google.message_id" in notification.data) {
+                        messageId = notification.data["google.message_id"];
```

Comment:
If this depends fully on the platform, then it might be cleaner to do `const messageId = Platform.OS === 'ios' ? notification.data[xxx] : notification.data[yyy];`

Reply:
Thanks! We've added this platform check in the latest commit.